metric/doc.go

```diff
@@ -13,11 +13,23 @@
 // limitations under the License.
 
 /*
-Package metric provides an implementation of the metrics part of the
-OpenTelemetry API.
+Package metric provides the OpenTelemetry API used to measure metrics about
+source code operation.
 
-This package is currently in a pre-GA phase. Backwards incompatible changes
-may be introduced in subsequent minor version releases as we work to track the
-evolving OpenTelemetry specification and user feedback.
+This API is separate from its implementations to allow for the reusability of
+the instrumentation built from it. See [go.opentelemetry.io/otel/sdk/metric]
+for the official OpenTelemetry implementation of this API.
+
+All measurements made with this package are made via instruments. These
+instruments are created by a [Meter] which itself is created by a
+[MeterProvider]. Applications need to accept a [MeterProvider] implementation
```

Comment:
```suggestion
[MeterProvider]. Applications or packages need to use a [MeterProvider] implementation
```

Reply:
This makes it sound like applications need to setup their own implementation instead of accept one from the operator of the code.

Reply:
Sure, use doesn't mean set or create, but it also doesn't communicate where the MeterProvider is created.

When we say the MeterProvider implementation needs to be accepted by the instrumentation it better conveys the creation of the implementation is external to the instrumentation.

Using use here introduces ambiguity and an valid interpretation would be that they think they need to setup their own implementation.

Reply:
Let's keep it as it is.